src/hex_tarball.erl

```diff
@@ -13,6 +13,33 @@
     {<<"Makefile">>, <<"make">>},
     {<<"Makefile.win">>, <<"make">>}
 ]).
+
+-define(DEPRECATED_META_FIELDS, [
+            <<"maintainers">>,
+            <<"contributors">>
```

Comment:
please use 4 whitespace indent (we're not super consistent with that, but that's we use most often)

Reply:
Will do.